app/main/views/suppliers.py

```diff
@@ -430,3 +435,31 @@ def update_supplier_framework_details(supplier_id, framework_slug):
         return jsonify(message="Database Error: {0}".format(e)), 400
 
     return jsonify(frameworkInterest=interest_record.serialize()), 200
+
+
+@main.route("/suppliers/<supplier_id>/briefs/<brief_id>", methods=["HEAD"])
+def supplier_eligible_for_brief(supplier_id, brief_id):
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).filter(
+        Brief.status == "live"
+    ).first_or_404()
+
+    supplier = Supplier.query.filter(
+        Supplier.supplier_id == supplier_id
+    ).first_or_404()
+
+    services = filter_services(
+        framework_slugs=[brief.framework.slug],
+        statuses=["published"],
+        lot_slug=brief.lot.slug,
+        location=brief.data["location"],
+        role=brief.data["specialistRole"] if brief.lot.slug == "digital-specialists" else None
+    )
+
+    services = services.filter(Supplier.supplier_id == supplier.supplier_id)
+
+    if len(services.all()):
+        return "", 200
```

Comment:
Is the apiclient going to be ok with an empty response? I think our default request methods try to parse the response as JSON?